code/modules/shuttle/battlecruiser_starfury.dm

```diff
@@ -178,4 +178,12 @@
 					source = spawner,
 				)
 
-	priority_announce("Unidentified armed ship detected near the station.")
+	// If we have a Station Trait "Naval Patrol" it will not prevent the Lone Ops but just announce it presence
+	// You should've secured that disk! Not even Navy will stop a disk-sniffing-determined Lone Operative
```

Comment:
The comment still mentions Lone Ops even though this is a announcement about a battlecruiser.

Reply:
Oh thank you